docker-compose.yml

```diff
@@ -29,6 +29,71 @@ services:
     networks:
       - io-fn
 
+  functions-node-20:
+    image: functions-node-20
+    build:
+      context: ./
+      dockerfile: ./docker/functions-node-20/Dockerfile
+    command: /bin/true
+
+  io-lollipop:
+    image: io-lollipop
+    build:
+      context: ./
+      dockerfile: ./docker/io-lollipop/Dockerfile
+    env_file:
+      - ./docker/generated/.env.io-lollipop
+    ports:
+      - ${FUNCTION_LOLLIPOP_PORT}:7071
+    networks:
+      - io-fn
+    depends_on:
+      - functions-node-20
+      - traefik
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.functions.rule=Host(`localhost`)"
+      - "traefik.http.routers.functions.entrypoints=web"
+      - "traefik.http.services.functions.loadbalancer.server.port=7071"
+
+      - "traefik.http.middlewares.testHeader.headers.customrequestheaders.x-user-id=unused"
+      - "traefik.http.middlewares.testHeader.headers.customrequestheaders.x-user-groups=${REQ_USER_GROUPS}"
+      - "traefik.http.middlewares.testHeader.headers.customrequestheaders.x-subscription-id=${REQ_SERVICE_ID}"
+      - "traefik.http.middlewares.testHeader.headers.customrequestheaders.x-user-email=${SERVICE_EMAIL}"
+      - "traefik.http.middlewares.testHeader.headers.customrequestheaders.x-user-note=unused"
+      - "traefik.http.middlewares.testHeader.headers.customrequestheaders.x-functions-key=unused"
+
+      # apply middleware to route
+      - "traefik.http.routers.functions.middlewares=testHeader"
+
+  traefik:
```

Comment:
do we need traefik here? at the moment, with the interoperability with io-mock project this can cause conflict.
cc. @gquadrati @BurnedMarshal

Reply:
No, we can remove it from now